mta/utils/ftp.py

```diff
@@ -591,7 +591,11 @@ def __init__(self, entity_path=None, entrypoint=None, host=None, login=None, pas
         )
 
         # Change working directory as per entity_path if provided
-        self.cwd(os.path.join(self.entrypoint, self.entity_path if entity_path else ""))
+        path = os.path.join(self.entrypoint, self.entity_path if entity_path else "")
+        # In windows OS adds forward slashes to the path , we need to replace them
+        new_path = path.replace("\\", "/")
```

Comment:
I think here better to add a check for the platform

Reply:
Done